guarddog/analyzer/sourcecode/clipboard-paste.yml

```diff
@@ -0,0 +1,10 @@
+rules:
+  - id: clipboard-paste
+    languages:
+      - python
+    message: This package has a potential misuse of pyperclip.paste()
+    metadata:
+      description: Identify when the 'pyperclip.paste()' command is executed
+    patterns:
+      - pattern: pyperclip.paste()
```

Comment:
if I understand correctly, `pyperclip.paste()` is accessing data in the keyboard - if that's the case, shouldn't we also match on `pyperclip.copy` (writing data to it)?